packages/devtools_app/lib/src/shared/theme.dart

```diff
@@ -432,6 +435,12 @@ extension ThemeDataExtension on ThemeData {
         fontSize: defaultFontSize,
       );
 
+  TextStyle get selectedLinkTextStyle => TextStyle(
```

Comment:
this can return `linkTextStyle.copyWith(color: colorScheme.devtoolsSelectedLink)`